file-upload/file-upload-scala-sbt/fileupload-impl/src/test/scala/com/example/fileupload/impl/FileuploadServiceSpec.scala

```diff
@@ -42,10 +42,16 @@ class FileUploadServiceSpec extends AsyncWordSpec with Matchers with BeforeAndAf
 
     }
     "handle file uploads " in {
+      val env = server.application.environment
 
       // Create a java.nio.file.Path to the file we want to upload
-      val originalPath: Path = new File(getClass.getResource("/sampleFile.txt").getPath).toPath
-      assert(originalPath.toFile.exists())
+      val maybePath: Option[Path] = env.resource("sampleFile.txt").map { url =>
+        new File(url.getPath).toPath
+      }
+      assert(maybePath.nonEmpty)
+
+      // Safe to do a get since the assertion above was successful
+      val originalPath = maybePath.get
```

Comment:
You can use `.getOrElse(throw ..)` to do with without having to justify the `.get` usage.